app/javascript/stylesheets/tailwind.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  purge: [
+    "./app/**/*.html.erb",
+    "./app/helpers/**/*.rb",
+    "./app/javascript/**/*.js",
+  ],
+  darkMode: false, // or 'media' or 'class'
```

Comment:
:( but we love dark mode!

Reply:
Haha we can add it later. Leaving it out now makes the file size smaller.

Reply:
haha which file size?

Reply:
If you have `darkMode: 'media'` or `darkMode: 'class'`, Tailwind loads more classes hence the filesize of it is bigger.